libqtile/core/manager.py

```diff
@@ -72,24 +71,17 @@ def _import_module(module_name, dir_path):
     return module
 
 
-def handle_exception(loop, context):
-    if "exception" in context:
-        logger.error(context["exception"], exc_info=True)
-    else:
-        logger.error("exception in event loop: %s", context)
-
-
 class Qtile(CommandObject):
```

Comment:
It looks like most of the changes to this class are to do the way it sets up its attributes. I am 100% down with these changes but would prefer they be separate from the other changes. Not necesarilly a separate PR, but maybe a preceding commit. Does that work with the other changes, or do some changes only make sense done in paralle with these?

Reply:
Almost all of them need to be done with the refactor since we can only add attributes to the class in __init__.

Reply:
OK, no worries then.